.github/workflows/nexus_integration_tests.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       run: |
         apt update && apt install -y git curl libclang-dev
     - name: Setup Rust for nexus_zenoh_bridge
-      uses: dtolnay/rust-toolchain@stable
+      uses: dtolnay/rust-toolchain@1.75
```

Comment:
We can remove this setup step entirely and instead rely on `rosdep` to install `cargo` as system dep version of `cargo` matches what we need.

Reply:
https://github.com/osrf/nexus/pull/35/commits/6cc59f376de6d1f13d303559366b726d4a80f68a